app/models/image_annotation.rb

```diff
@@ -16,7 +16,7 @@ class ImageAnnotation < Annotation
   def extract_coords
     horiz_range = { start: [x1, x2].min, end: [x1, x2].max }
     vert_range = { start: [y1, y2].min, end: [y1, y2].max }
-    { id: annotation_text_id, x_range: horiz_range, y_range: vert_range }
+    { id: annotation_text_id, annot_id: self.id, x_range: horiz_range, y_range: vert_range }
```

Comment:
This change breaks two tests in `test/unit/submission_file_test.rb`. Please update them and run the tests in this file (`rake test TEST=test/unit/submission_file_test.rb`) to make sure they're all okay.

Then repeat for `spec/models/image_annotation_spec.rb`.